tests/integration/test_arithmetic.py

```diff
@@ -112,7 +112,7 @@ def main(a1: angle, a2: angle) -> bool:
         a3 = -a1 + a2 * -3
         a3 -= a1
         a3 += 2 * a1
-        return a3 == -a2
+        return a3 / 3 == -a2
```

Comment:
This is a straightforward reversion of an unnecessary change. (The code is not executed, the result does not have to be `true`, and division is valid.)